python/build-module.py

```diff
@@ -17,6 +17,7 @@
 
 import os
 import shutil
+
```

Comment:
This line surprises me since they are both internal packages.

Reply:
https://beta.ruff.rs/docs/settings/#isort-lines-between-types
I can try setting it up 0